.github/dependabot.yml

```diff
@@ -753,10 +753,6 @@ updates:
     directory: /docker/duoadmin3
     schedule:
      interval: daily
-  - package-ecosystem: pip
```

Comment:
Was this deliberately removed?

Reply:
No, I believe this is a mistake. Will add these lines back